CondFormats/SiPixelTransient/src/SiPixelTemplate.cc

```diff
@@ -1483,120 +1489,103 @@ bool SiPixelTemplate::interpolate(int id, float cotalpha, float cotbeta, float l
-          yparh_[i][j] = thePixelTemp_[index_id_].enty[ihigh].ypar[i][j];
+          yparl_[i][j] = enty0_->ypar[i][j];
+          yparh_[i][j] = enty1_->ypar[i][j];
         }
-        if (flip_x) {
-          xparly0_[1 - i][j] = thePixelTemp_[index_id_].enty[ilow].xpar[i][j];
-          xparhy0_[1 - i][j] = thePixelTemp_[index_id_].enty[ihigh].xpar[i][j];
+        if (flip_x_) {
+          xparly0_[1 - i][j] = enty0_->xpar[i][j];
+          xparhy0_[1 - i][j] = enty1_->xpar[i][j];
         } else {
-          xparly0_[i][j] = thePixelTemp_[index_id_].enty[ilow].xpar[i][j];
-          xparhy0_[i][j] = thePixelTemp_[index_id_].enty[ihigh].xpar[i][j];
+          xparly0_[i][j] = enty0_->xpar[i][j];
+          xparhy0_[i][j] = enty1_->xpar[i][j];
         }
       }
     }
 
     for (i = 0; i < 4; ++i) {
       yavg_[i] = (1.f - yratio) * thePixelTemp_[index_id_].enty[ilow].yavg[i] +
                  yratio * thePixelTemp_[index_id_].enty[ihigh].yavg[i];
-      if (flip_y) {
+      if (flip_y_) {
         yavg_[i] = -yavg_[i];
       }
-      yrms_[i] = (1.f - yratio) * thePixelTemp_[index_id_].enty[ilow].yrms[i] +
-                 yratio * thePixelTemp_[index_id_].enty[ihigh].yrms[i];
-      //	      ygx0_[i]=(1.f - yratio)*thePixelTemp_[index_id_].enty[ilow].ygx0[i] + yratio*thePixelTemp_[index_id_].enty[ihigh].ygx0[i];
-      //	      if(flip_y) {ygx0_[i] = -ygx0_[i];}
-      //	      ygsig_[i]=(1.f - yratio)*thePixelTemp_[index_id_].enty[ilow].ygsig[i] + yratio*thePixelTemp_[index_id_].enty[ihigh].ygsig[i];
-      //	      xrms[i]=(1.f - yratio)*thePixelTemp_[index_id_].enty[ilow].xrms[i] + yratio*thePixelTemp_[index_id_].enty[ihigh].xrms[i];
-      //	      xgsig[i]=(1.f - yratio)*thePixelTemp_[index_id_].enty[ilow].xgsig[i] + yratio*thePixelTemp_[index_id_].enty[ihigh].xgsig[i];
-      chi2yavg_[i] = (1.f - yratio) * thePixelTemp_[index_id_].enty[ilow].chi2yavg[i] +
-                     yratio * thePixelTemp_[index_id_].enty[ihigh].chi2yavg[i];
-      chi2ymin_[i] = (1.f - yratio) * thePixelTemp_[index_id_].enty[ilow].chi2ymin[i] +
-                     yratio * thePixelTemp_[index_id_].enty[ihigh].chi2ymin[i];
-      chi2xavg[i] = (1.f - yratio) * thePixelTemp_[index_id_].enty[ilow].chi2xavg[i] +
-                    yratio * thePixelTemp_[index_id_].enty[ihigh].chi2xavg[i];
-      chi2xmin[i] = (1.f - yratio) * thePixelTemp_[index_id_].enty[ilow].chi2xmin[i] +
-                    yratio * thePixelTemp_[index_id_].enty[ihigh].chi2xmin[i];
-      yavgc2m_[i] = (1.f - yratio) * thePixelTemp_[index_id_].enty[ilow].yavgc2m[i] +
-                    yratio * thePixelTemp_[index_id_].enty[ihigh].yavgc2m[i];
-      if (flip_y) {
+      yavg_[i] = (1.f - yratio) * enty0_->yavg[i] + yratio * enty1_->yavg[i];
+      if (flip_y_) {
+        yavg_[i] = -yavg_[i];
+      }
+      yrms_[i] = (1.f - yratio) * enty0_->yrms[i] + yratio * enty1_->yrms[i];
+      //	      ygx0_[i]=(1.f - yratio)*enty0_->ygx0[i] + yratio*enty1_->ygx0[i];
```

Comment:
Is this commented out code needed: add one line explaining its usage, if so; otherwise just remove

Reply:
Removed